internal/runners/clean/run_win.go

```diff
@@ -117,7 +119,8 @@ func removePaths(logFile string, paths ...string) error {
 
 	args := []string{"/C", sf.Filename(), logFile, fmt.Sprintf("%d", os.Getpid()), filepath.Base(exe)}
 	args = append(args, paths...)
-	_, err = exeutils.ExecuteAndForget("cmd.exe", args)
+
+	_, err = exeutils.ExecuteAndForget(subshell.DetectShellBinary(cfg), args)
```

Comment:
Should revert this. The flags are specifically for cmd.exe. And the issue of not finding cmd.exe on path was a symptom, not the problem we're fixing.